Kernel/API/SyscallString.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <AK/StringView.h>
+#include <Kernel/API/Syscall.h>
+
+namespace Kernel {
+namespace Syscall {
```

Comment:
Please use C++17 nested namespaces

```suggestion
namespace Kernel::Syscall {
```